packages/ckeditor5-clipboard/src/dragdroptarget.ts

```diff
@@ -122,16 +123,23 @@ export default class DragDropTarget extends Plugin {
 		targetViewRanges: Array<ViewRange> | null,
 		clientX: number,
 		clientY: number,
-		blockMode: boolean
+		blockMode: boolean,
```

Comment:
Nitpick: There are so many arguments. I'd consider wrapping it with `option` argument. That way on call site the arguments would be named, not positional and less error-prone.

Reply:
This would be a breaking change because this method is public and this PR will land in the next release (after the one where we'll introduce the `DragDropTarget` class).

Also, to be consistent, we should also change the params in the `getFinalDropRange` method, which would introduce a lot of noise in the PR.

Unless this is a hard blocker, I would skip this change.

Reply:
It is not public, but internal. My comment was a nitpick - safe to be ignored.